_data/projects/libvlc-go.yml

```diff
@@ -0,0 +1,13 @@
+name: libvlc-go
+desc: Golang bindings for libvlc 2.X/3.X/4.X used by the VLC media player
+site: https://github.com/adrg/libvlc-go
+tags:
+- Go
+- Golang
+- VLC
+- Video
+- Sound
+- Bindings
+upforgrabs:
+  name: help wanted
+  link: https://github.com/adrg/libvlc-go/labels/help%20wanted
```

Comment:
@adrg I noticed that your repo doesn't have any issues related to this tag. Please add some so that I can move with merging it.

Reply:
@ritwik12 Added some issues with the above mentioned label.